fpdc/components/migrations/0002_modules.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.18 on 2019-02-06 10:21
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("components", "0001_initial")]
+
+    operations = [
+        migrations.CreateModel(
+            name="Module",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True, primary_key=True, serialize=False, verbose_name="ID"
+                    ),
+                ),
+                ("name", models.CharField(max_length=255, unique=True)),
+                ("point_of_contact", models.CharField(max_length=100)),
```

Comment:
You don't specify blank=False here?

Reply:
This is generated by the Django ORM, blank=False is the default (https://docs.djangoproject.com/en/2.1/ref/models/fields/#blank) so it does not have to be explicit in the migration, however I think it is good to make it explicit in the models.py so when you read the code it is clear.